docs/TimelineGuide.md

```diff
@@ -3,12 +3,14 @@
 This is a guide for people who want to write timelines,
 primarily for cactbot.
 
-NOTE: this guide is a good bit out of date:
+NOTE: When this guide was originally written,
+Cape Westwind was a standard 8-player trial.
+This was removed in Endwalker, and that version of the encounter is no longer accessible.
+However, the timeline creation process for it is still valid,
```

Comment:
This doesn't have to be done in this PR, but I think it would be useful to pick a fight (it doesn't have to be Cape Westwind) that can have an example log from content that people can run solo (unsynced) such that they can use the example log to make/test.

I think the Cape Westwind example is a little outdated, personally. I think an example that shows off as many of these as possible would be good:
* starting from in combat
* at least one loop
* autos that should be ignored
* multiple abilities, e.g. left OR right cleave
* maybe abilities that should be commented out for noisiness (like exaflare hits or something)

Reply:
Agreed. I would have liked to have something like that to go along with the rest of this PR, but it's a *lot* of work to build that all out and I wasn't really in a place to do that here. I'm hoping that with just "replace the timeline to something modern" outstanding it will be easier to get that change made.

Reply:
Oh yeah, for sure it's a lot of work. That's why I haven't done it either. Just thinking about the future, but this is a really good change on its own to not mislead people.